src/Entries/EntryRepository.php

```diff
@@ -55,6 +55,8 @@ public function save($entry)
         $model->save();
 
         $entry->model($model->fresh());
+
+        Blink::put("eloquent-entry-{$entry->id()}", $entry);
```

Comment:
Probably want to also do it for `"eloquent-entry-{$entry->uri()}"` as per https://github.com/statamic/eloquent-driver/blob/2c4a775accb30803120b009209bc80ebcca23093/src/Entries/EntryRepository.php#L38-L41

Reply:
Yep - thats done